packages/rum-core/src/domain/rumEventsCollection/view/viewCollection.spec.ts

```diff
@@ -23,8 +23,14 @@ describe('viewCollection', () => {
 
   it('should create view from view update', () => {
     const { lifeCycle, rawRumEvents } = setupBuilder.build()
+    const location: Partial<Location> = {}
     const view = {
+      location,
```

Comment:
```suggestion
      location: location as Location,
```
it would remove the need to cast view as View in the next statement